Comienza el simon dice

AdriCappelletti opened the pull request:
Agrega funciones basicas para iniciar el juego
pero todavia no valida si ambos arrays son iguales

Commit:
Comienza el simon dice

Agrega funciones basicas para iniciar el juego
pero todavia no valida si ambos arrays son iguales

fsodano commented on JS/app.js:
```diff
@@ -1,12 +1,80 @@
-const square2 = document.querySelector('#cuadro2')
-const square3 = document.querySelector('#cuadro3')
-const square4 = document.querySelector('#cuadro4')
+// const square1 = document.querySelector("#cuadro1");
+// const square2 = document.querySelector("#cuadro2");
+// const square3 = document.querySelector("#cuadro3");
+// const square4 = document.querySelector("#cuadro4");
```
borrá esto si no lo necesitás

fsodano commented on JS/app.js:
```diff
@@ -1,12 +1,80 @@
+// const square2 = document.querySelector("#cuadro2");
+// const square3 = document.querySelector("#cuadro3");
+// const square4 = document.querySelector("#cuadro4");
+const bt = document.querySelector("#start-round");
```
bt es un mal nombre

fsodano commented on JS/app.js:
```diff
@@ -1,12 +1,80 @@
+};
+
+function machineTurn() {
+  round++;
+  roundNumber.textContent = round;
+  state.textContent = "Machine turn";
```
esto va en el roundHandler

fsodano commented on JS/app.js:
```diff
@@ -1,12 +1,80 @@
+
+function roundHandler() {
+  machineTurn();
+  const PLAYER_DELAY = (machineMovements.length + 1) * 1000;
```
esto estaría mejor en una función, por otro lado no es una constante con mayúscula porque no es un valor estático, o sea, no es algo onda `PLAYER_DELAY = 1;`

fsodano commented on JS/app.js:
```diff
@@ -1,12 +1,80 @@
+      highlight(square);
+      let n = square.getAttribute("data-number");
+      userMovements.push(n);
+      if (compareArrays(userMovements, machineMovements) === true) {
```
tenés que considerar cuándo querés comparar los arrays (no es cada vez que el jugador hace un click)

Commit:
Arregla errores y termina el juego

Arregla errores de PR anteriores y termina el juego.
Agrega comparacion de ambos jugadores y desabilita el input del
usuario cuando juega la maquina.
Agrega una funcion que muestra un cartel cuando el jugador pierde
y le permite comenzar de nuevo.
